rotkehlchen/chain/manager.py

```diff
@@ -811,7 +812,16 @@ def modify_blockchain_accounts(
             self.flush_cache('query_balances', arguments_matter=True)
             self.flush_cache('query_balances', arguments_matter=True, blockchain=SupportedBlockchain.ETHEREUM)  # noqa: E501
             for account in accounts:
-                address = deserialize_ethereum_address(account)
+                try:
+                    address = deserialize_ethereum_address(account)
+                except DeserializationError as e:
+                    log.error(f"Error deserializing eth address {account}")
+                    # As suggested by @vnavascues `DeserializationError` is not in the errors
```

Comment:
Actually no this is not a good idea.

RemoteError means that there was something wrong with a remote and is caught and handled by whoever calls this function.

If you grep the code you will see this function, for ethereum accounts, is called only when you add or when you remove a blockchain account. This is an action coming from the api. And the api has input sanitization for this (and other) input. See here: https://github.com/rotki/rotki/blob/3de2916cdd5cc012f7819d678157d1dc56f9cda0/rotkehlchen/api/v1/encoding.py#L369-L396

So by this state you should expect it's already a valid ethereum address. So remove all this and just use the `string_to_xxx` function.

What you can do is note this down in the function's docstring (that this should be called with valid ethereum accounts) for future readers of the code.